src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/CourtReport.java

```diff
@@ -0,0 +1,119 @@
+package uk.gov.justice.digital.delius.jpa.standard.entity;
+
+import lombok.Data;
+import lombok.ToString;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+
+@Data
+@Entity
+@Table(name = "COURT_REPORT")
+@ToString(exclude="courtAppearance")
+public class CourtReport {
+    @Id
+    @Column(name = "COURT_REPORT_ID")
+    private Long courtReportId;
+
+    @Column(name = "COURT_APPEARANCE_ID")
+    private Long courtAppearanceId;
+
+    @ManyToOne
+    @JoinColumn(name = "COURT_APPEARANCE_ID", referencedColumnName="COURT_APPEARANCE_ID", insertable = false, updatable = false)
```

Comment:
Possibly delete field courtAppearanceId if not needed elsewhere, and remove the insertable & updateable attributes

Reply:
Yes not needed so deleted